src/OpenLoco/Objects/IndustryObject.h

```diff
@@ -51,16 +51,16 @@ namespace OpenLoco
         uint16_t nameSingular;        // 0x0A
         uint16_t namePlural;          // 0x0C
         uint32_t var_0E;
-        uint32_t var_12;
+        uint32_t var_12; // Base image id for building 0
```

Comment:
any reason not to rename the variable here (and below)?

Reply:
I'm not 100% sure at this time. I really need to do the paint function before being confident.

Reply:
sounds fine to me